#exception handling

# try:
#     name="narendra"
#     age=24
#     print(name,hight)
# except Exception as e :
#     print("ok just run it")
# finally:
#     a=100
#     b=23
#     print(a+b)
# def func():
#     try:
#         pen=input("enter name")
#         paper=int(input("enter age"))
#     except:
#         print()
      
#     finally:
#         print()
# func()     
                

# list=[20,20,12,12,344,567,235,]
# name=list()
      
     